integration/src/main/java/scripts/datageneration/regenerate_perf_data.sql

```diff
@@ -0,0 +1,47 @@
+-- This function inserts a semi-random linked account and passport with
+-- the given user_id and external_user_id
+CREATE PROCEDURE insert_test_record(user_id text, external_user_id text)
+LANGUAGE SQL
+AS $$
+  -- Insert a linked account with the given values
+  INSERT INTO linked_account
+  (id, user_id, provider_name, refresh_token, expires, external_user_id, is_authenticated)
+  VALUES (
+    nextval('linked_account_id_seq'),
+    user_id,
+    'ras',
+    'testToken',
+    current_timestamp + interval '3000 year',
+    external_user_id,
+    true
+  );
+  -- Insert a corresponding passport
+  INSERT INTO ga4gh_passport (linked_account_id, jwt, expires)
+  VALUES (
+    currval('linked_account_id_seq'),
+    'testJwt',
```

Comment:
make this a more realistic jwt. I am especially concerned that it is a realistic size.

Reply:
I looked into a few different options for doing this but postgresql is pretty limited in what it can actually do:
1. You had suggested reading a static value from a file. Unfortunately, there doesn't seem to be a way to read from a file path relative to the script location. Instead, it reads relative to the location of the postgres data directory, which isn't helpful. 
2. If we want an actual JWT, we could just hard code the JWT string in the SQL script. This is definitely the simplest solution, even though I do prefer to avoid hard-coding things. 
3. If we only care about the string being a larger size and want to avoid hard coding, we could randomly generate a long string by doing something like [this](https://www.simononsoftware.com/random-string-in-postgresql/).

I would love to hear your thoughts on what you think is best!

Reply:
I ended up coming up with something that I think works pretty well to randomly generate a long string. Let me know what you think!